### DISCUSSION TITLE
Power management systems for battery

### DISCUSSION THREAD

**Discussion by pablo-collantes** [Comet-Robotics/URC#37] [PCB/Power]
Our battery is expected to be 7s6p, right now were ordering 21 cells for 7s3p for capacity testing.
on a 12 volt system out motors pulled 55 amps at peak but on flat terrain consumed closer to 15 amps.
On 24v system amp consumption cut in half.
So maximum continuous even on an incline will most likely be under 60 amps from the batteries.

What we need:
A 24 to 12 volt DC to DC converter:
- This will most likely have to handle high peak output current on roughly 60 amps, probably safer to find one that can handle 60 amp continuous.
- Be weary of very cheap converters try to cross examine them online to see is they're are actually legit.

A Battery Management System (BMS):
- Probably will go with this BMS if Robosub says this brand isn't bad [BMS](https://www.amazon.com/DALY-Protection-Battery-Management-Current/dp/B08741XNDG/ref=asc_df_B08741XNDG?mcid=9af3883393643b3f891d61b02aa46c37&tag=hyprod-20&linkCode=df0&hvadid=693418895745&hvpos=&hvnetw=g&hvrand=456913692840395737&hvpone=&hvptwo=&hvqmt=&hvdev=c&hvdvcmdl=&hvlocint=&hvlocphy=9026948&hvtargid=pla-981541710657&th=1)
- If not, prepare an over discharge protection board which entails a single voltage sensor and a bunch of MOSFETS to change which cell is chosen.

Also if we don't trust the BMS at least for discharge we can still go and buy a very cheap 7s 10 amp bms that can handle charge balancing

TLDR:
- Look for a suitable 24v - 12v DC to DC converter that can handle 60 amp continuous output.
- Double check DALY BMS board, if bad, design a simple over discharge protection board so cells don't drop below 2.5v and find cheap 7s BMS for charging
